CMakeLists.txt

```diff
@@ -47,6 +47,11 @@ include (src/cmake/utilities.txt)
 # Build configurations.
 #--------------------------------------------------------------------------------------------------
 
+# Disable in source builds.
```

Comment:
Out of curiosity, is there a particular reason to forbid it? Not that it's a good idea to do in-source builds...